src/projects.html

```diff
@@ -1,87 +1,143 @@
 --- 
-layout: default 
-title: Projects | Women Hack for Non-Profits
+layout: default
+title: Projects | Women Hack for Non-Profits 
 ---
-    <section class="smaller-banner projects-image">
-      <div class="content">
+<section class="smaller-banner projects-image">
+    <div class="content">
         <h1 class="title-heading">Our Projects</h1>
-      </div>
-    </section>
-    <section>
-      <div class="content">
-        <h2 class="section-heading">Completed Projects </h2>
+    </div>
+</section>
+<section>
```

Comment:
I wonder whether you will need a class on that section to later on style this section in particular.